spinnaker/api/pipeline.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 func CreatePipeline(client *gate.GatewayClient, pipeline interface{}) error {
-	resp, err := client.PipelineControllerApi.SavePipelineUsingPOST(client.Context, pipeline)
+	resp, err := client.PipelineControllerApi.SavePipelineUsingPOST(client.Context, pipeline, nil)
```

Comment:
Could you pass the option instead of `nil`, please? See details here https://github.com/spinnaker/spin/blob/c603448a907164b79789cd9afc41c527c0b03a1d/cmd/pipeline/update.go#L88

Same thing to other API calls.

Reply:
Fixed